src/hdusd/engine/viewport_engine.py

```diff
@@ -264,11 +264,14 @@ def draw(self, context):
         self.render_params.renderResolution = (view_settings.width, view_settings.height)
         self.render_params.clipPlanes = [Gf.Vec4d(i) for i in gf_camera.clippingPlanes]
 
-        if self.shading_data.type == 'MATERIAL' and self.is_gl_delegate:
+        if self.is_gl_delegate:
             l = Glf.SimpleLight()
             l.ambient = (0, 0, 0, 0)
             l.position = (*gf_camera.frustum.position, 1)
 
+            if not self.shading_data.type == 'MATERIAL':
```

Comment:
Still have red light in viewport material preview.
Also after switching from material preview to rendered red light remains.
Please fix this.